lib/declaration.ts

```diff
@@ -439,6 +446,22 @@ class DeclarationTranspiler extends base.TranspilerBase {
       }
     }
   }
+
+  private visitCallSignature(name: string, signature: ts.SignatureDeclaration,
+                             typeParameters: ts.NodeArray<ts.TypeParameterDeclaration>) {
+    this.emit('typedef');
+    if (signature.type) {
```

Comment:
This is the return type in the signature. The only things I'm hoisting in from the interface declaration are the generic type parameters and the name. My tests wouldn't pass otherwise, I think.